### DISCUSSION TITLE
cant connect connection mysql

### DISCUSSION THREAD

**Discussion by rezaprtma391** [leokhoa/laragon#838] [General]
![Cuplikan layar 2024-07-20 202656](https://github.com/user-attachments/assets/d520c5ba-2618-4876-a003-916b7a0832c7)

Please help me why?

i do update mysql on laragon and start 
![Cuplikan layar 2024-07-20 202852](https://github.com/user-attachments/assets/bc1d4fd1-3220-4302-8aea-a134a965f50b)

**Comment by luisAntonioLAGS** [comment]
Laragon needs extra configurations to run mysql 8.4 and mysql 9.
The topic has already been discussed in the following thread: https://gist.github.com/meorajrul/b57803bf1b4ddfd2f93e6ad37c3ac5f2
